docs/api-docs/api-use-cases.md

```diff
@@ -0,0 +1,300 @@
+# API Use Cases
+
+This document outlines use cases for APIs at BICAN portals. 
+Requirements supporting the use cases are documented in a separate [requirements doc](api-requirements.md).
+
+
+## Read use cases
+The following use cases describe scenarios where a downstream system would read from a given system's API.
```

Comment:
I know we are planning on building a REST interface, but one advantage of GraphQL is that all of these "Read" use cases would just be one endpoint and one schema to maintain.

Reply:
Sure, none of these requirements mandate a strict RESTful API. The examples follow that paradigm, but GraphQL is fine as well.